pre_commit/util.py

```diff
@@ -268,3 +271,70 @@ def handle_remove_readonly(
 def parse_version(s: str) -> Tuple[int, ...]:
     """poor man's version comparison"""
     return tuple(int(p) for p in s.split('.'))
+
+
+def in_docker() -> bool:
+    """
+    Check if running in Docker
+    :return: Whether or not this is running in Docker container
+    """
+    try:
+        with open('/proc/1/cgroup') as cgroup_file:
+            return 'docker' in cgroup_file.read()
+    except FileNotFoundError:
+        return False
+
+
+def translate_path(path: str) -> str:
+    """
+    Method to get the right path considering it can be mounted in Docker
+     already.
+    :param path: A string representing a path within the container
+    :return: A string representing a path on the host (or the original
+             path if the path is not in a bound volume)
+    """
+    if not in_docker():
+        return path
+    binds = get_binds()
+    if path in binds.keys():
+        return binds[path]
+    exps = ['(%s)/(.*)' % key for key in binds.keys()]
```

Comment:
[never use `.keys()` -- iterating over a dict works as-is](https://github.com/asottile/scratch/wiki/PythonHintsForNoobs#iterating-a-dict-gives-you-its-keys)